### ISSUE TITLE
A suggestion for your teaching style

### ISSUE THREAD

**Issue by byers174** [btholt/complete-intro-to-react-v5#42]
Perhaps I'm out of place as a newcomer, but I have a suggestion I think you should take into account for "intro" courses.

You are clearly an expert, and so you take a lot of shortcuts when coding. Unfortunately, this causes me to get confused. For example (one of many), you destructure code to make it look more elegant. The cost is that I'm now worrying about learning shortcut concepts like this (and the others) instead of focusing on react, the entire reason I'm here. I'm having to pause just to read over the code going "okay, what did I just write? How would this look normally [to me]," and then refactor it without your shortcuts in order to digest the react information such as how useEffect works. I'm determined, but it is a deterrent. 

I could understand using this in intermediate courses, or having a mini JS shortcuts lesson at the very beginning with in-depth examples (opposed to showing something once at full-speed) but I think it would be beneficial for any "into-to" course, which is designed for beginners, to limit your shortcuts, tricks, and elegant code, unrelated to the core concept, so that users may focus on what they came for. 

tldr, I am unable to immediately absorb your teachings on react pieces like useEffect because I'm having to unpack your JS/JSX shortcuts first. 

Again, I admit to being new, and I understand the purpose and effectiveness of your shortcuts (and even the shortcuts themselves with outside materials)... it is just personally discouraging to have to stop so often on an intro course because I am only used to seeing long-handed explicit code. Something that would work for me personally is a 'refactor' episode at the end where all of these things can be implemented, showing us how to save time after the material has been learned. I am learning a lot however and I would like to thank you for teaching this course.
